cwbi-auth-http-client/src/main/java/hec/army/usace/hec/cwbi/auth/http/client/TokenRequestBuilder.java

```diff
@@ -1,13 +1,10 @@
 package hec.army.usace.hec.cwbi.auth.http.client;
 
-import hec.army.usace.hec.cwbi.auth.http.client.token.fluentbuilders.RequestClientId;
-import hec.army.usace.hec.cwbi.auth.http.client.token.fluentbuilders.TokenRequestExecutor;
-import hec.army.usace.hec.cwbi.auth.http.client.token.fluentbuilders.TokenRequestFluentBuilder;
 import java.io.IOException;
 import java.util.Objects;
 import mil.army.usace.hec.cwms.http.client.auth.OAuth2Token;
 
-abstract class TokenRequestBuilder implements TokenRequestFluentBuilder {
+public abstract class TokenRequestBuilder implements TokenRequestFluentBuilder {
```

Comment:
Did you intend to have this class sub-classed from other packages?

Reply:
No, this is intended to act as a base class used internally by RefreshTokenRequestBuilder and DirectGrantX509RequestBuilder (which is why it was package-private initially). But the fluent api required the overridden url method be public, so callers from outside the class can use the builder as intended. So I think, the initial package-private class is better, since the implementing classes are public?

Reply:
This should have been your response previously.